src/GTP.cpp

```diff
@@ -327,7 +432,7 @@ bool GTP::execute(GameState & game, std::string xinput) {
     } else if (command.find("komi") == 0) {
         std::istringstream cmdstream(command);
         std::string tmp;
-        float komi = 7.5f;
+        float komi = cfg_target_komi;
         float old_komi = game.get_komi();
 
         cmdstream >> tmp;  // eat komi
```

Comment:
I think we can just process this now and replace cfg_target_komi with it?

Reply:
Currently the command line parameter --target-komi is used for m_stm_komi, m_opp_komi (both for the NN) and for m_komi (for scoring), but the `komi` command only changes m_komi. For now it's good to be possible to specify different komi for scoring and for NN, since the NN doesn't perceive komi accurately and undistortedly.

Reply:
For compatibility it might be good to derive a best guess value for m_stm_komi and m_opp_komi when "komi" is set this way.